### PR TITLE
[Feature] Add clickhouse docs

### LINKED ISSUE #5: [FEATURE] Integration with ClickHouse
**Is your feature request related to a problem? Please describe.**

[ClickHouse](https://clickhouse.com/) is the fastest and most resource efficient open-source database for real-time apps and analytics with full SQL support and a wide range of functions to assist users in writing analytical queries. Lately added data structures and distance search functions (like cosineDistance) as well as [approximate nearest neighbor search indexes](https://clickhouse.com/docs/en/engines/table-engines/mergetree-family/annindexes) enable ClickHouse to be used as a high performance and scalable vector database to store and search vectors with SQL.

**Describe the solution you'd like**

Add ClickHouse as embedding store integration

**Describe alternatives you've considered**

No

**Additional context**

No

### PR DESCRIPTION
**Pull request by Martin7-1**
## Issue

Closes [langchain4j-community #5](https://github.com/langchain4j/langchain4j-community/issues/5)

## Change

Add `langchain4j-community-clickhouse` document.

## General checklist

- [x] There are no breaking changes
- [ ] I have added unit and integration tests for my change
- [ ] I have manually run all the unit and integration tests in the module I have added/changed, and they are all green
- [ ] I have manually run all the unit and integration tests in the [core](https://github.com/langchain4j/langchain4j/tree/main/langchain4j-core) and [main](https://github.com/langchain4j/langchain4j/tree/main/langchain4j) modules, and they are all green

- [x] I have added/updated the [documentation](https://github.com/langchain4j/langchain4j/tree/main/docs/docs)
- [ ] I have added an example in the [examples repo](https://github.com/langchain4j/langchain4j-examples) (only for "big" features)
- [ ] I have added/updated [Spring Boot starter(s)](https://github.com/langchain4j/langchain4j-spring) (if applicable)

### CODE REVIEW

File: docs/docs/integrations/embedding-stores/clickhouse.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+enable ClickHouse to be used as a high performance and scalable vector database to store and search
+vectors with SQL.
+
+## Maven Dependency
+
+```xml
+
+<dependency>
+    <groupId>dev.langchain4j</groupId>
+    <artifactId>langchain4j-community-elasticsearch</artifactId>
```
Comment (langchain4j) [inline docs/docs/integrations/embedding-stores/clickhouse.md]:
```suggestion
    <artifactId>langchain4j-community-clickhouse</artifactId>
```

Should it be clickhouse?

**Review by ghost — COMMENTED** [review]
@Martin7-1 thank you!

File: docs/docs/integrations/embedding-stores/clickhouse.md (hunk 1 above)
Comment (Martin7-1) [reply docs/docs/integrations/embedding-stores/clickhouse.md]: Of course...My bad to just copy it.

**Review by ghost — APPROVED** [review]
@Martin7-1 thank you!

**Pull request merged**